CHANGELOG.md

```diff
@@ -0,0 +1,515 @@
+# Changelog
+
+All notable changes are documented in this file.
+The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` and `Removed`.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+## [5.0.0] - 2023-04-13
+
+### Added
+* Add extension changelog by @cd-work (#1019)
```

Comment:
Is there a reason why you went for `*` instead of `-` for the lists here? Seems rather unconventional to me.

Reply:
That is what is in our GitHub release notes, which this was generated from. And [Markdown doesn't care](https://spec.commonmark.org/0.30/#bullet-list-marker)... But I also don't care, so we can standardize on `-` since that seems to be more common.